streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -388,6 +401,40 @@ private String nodeSensorPrefix(final String threadId, final String taskId, fina
             + SENSOR_PREFIX_DELIMITER + SENSOR_NODE_LABEL + SENSOR_PREFIX_DELIMITER + processorNodeName;
     }
 
+    public Sensor topicLevelSensor(final String threadId,
+                                   final String taskId,
+                                   final String processorNodeName,
+                                   final String topicName,
```

Comment:
I cannot access the apache wiki atm (seems down temporarily) but just double checking is topic name part of the tag in the KIP proposal as well? If not we should update the wiki and the KIP thread.

Reply:
I did update it to include the `topic` tag already, but I haven't updated it yet to clarify that we are introducing these at a new "topic-level" metrics scope. I'll do that right now